source/index.ts

```diff
@@ -374,7 +374,7 @@ export class Conf<T extends any = any> implements Iterable<[keyof T, T[keyof T]]
 
 	private _handleChange<Key extends keyof T>(
 		getter: () => T | T[Key] | undefined,
-		callback: OnDidAnyChangeCallback<T> | OnDidChangeCallback<T[Key]>
+		callback: OnDidAnyChangeCallback<any> | OnDidChangeCallback<any>
```

Comment:
Why `any`?

Reply:
It was because it was hard to type `.call`. But I found a solution.